AppCenter/AppCenter/include/MSACAppCenterErrors.h

```diff
@@ -1 +1 @@
-../MSAppCenterErrors.h
\ No newline at end of file
+../MSACAppCenterErrors.h
```

Comment:
Could you please also add formatting for files to App Center core?

Reply:
Addressed in https://github.com/microsoft/appcenter-sdk-apple/pull/2180